app/celery/reporting_tasks.py

```diff
@@ -24,3 +25,5 @@ def create_nightly_billing(day_start=None):
 
         for data in transit_data:
             update_fact_billing(data, process_day)
+
+    current_app.logger.info("create-nightly-billing task complete. {} rows updated".format(len(transit_data)))
```

Comment:
Is this only logging the rows that were updated for the last day processed?

Reply:
Ah - yes you are right. I'm not really sure what logging we need here.

Reply:
Ideas?

Reply:
i'd probably move the log line in the for loop so you get three log messages - one for each day.